src/handlers/journey-event-error-handler.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  journeyEventErrorHandler(err, req, res, next) {
+    if (res.headersSent) {
+      return next(err);
+    }
+
+    res.err = err; // this is required so that the pino logger does not log new error with a different stack trace
+
+    if (res.err?.response?.data?.page) {
+      res.status(res.err.response.data.statusCode);
+      return res.render(`ipv/${res.err.response.data.page}.njk`);
```

Comment:
Feel like we should sanitise this input in some way? not sure what res.render does itself if anything

Reply:
Yeah we sanitize the res.render input elsewhere so i'll look at adding that here to